examples/opf/simple_server/model_params.py

```diff
@@ -0,0 +1,247 @@
+
+# Numenta Platform for Intelligent Computing (NuPIC)
+# Copyright (C) 2013, Numenta, Inc.  Unless you have an agreement
+# with Numenta, Inc., for a separate license for this software code, the
+# following terms and conditions apply:
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see http://www.gnu.org/licenses.
+#
+# http://numenta.org/licenses/
+# ----------------------------------------------------------------------
+
+MODEL_PARAMS = {
+    # Type of model that the rest of these parameters apply to.
+    'model': "CLA",
```

Comment:
Examples aren't on the PYTHONPATH so importing between them is hacky at best. I think it is a good idea to share the suggested parameters though. @tomsilver added something similar for making standard network configurations reusable:
https://github.com/numenta/nupic/blob/master/nupic/engine/common_networks/__init__.py